### PR TITLE
#11887 but remade  (plate armor) 

### PR DESCRIPTION
**Pull request by ghost**
#11887
:cl: zamolxius & not dichard but rick/Gorram_Swags
add: Added plate armor, welded from metal.
imageadd: added the sprites for the plate armor
/:cl:

## Why It's Good For The Game
Adds plate armor, it has negative resistance towards thermal damage sources like laser or cold, negative resistance towards energy because no insulation, it protects decently against radiation and bio because it's thick metal, but it slows you down. It has great resistance towards melee and bullet damage.

https://i.imgur.com/nPQQyEw.png
It can be spraypainted with spraypaint for color

### CODE REVIEW

**Comment by karmaisblackandbluepilled** [comment]
+1

**Comment by Majick1** [comment]
+1

**Comment by yoyobatty** [comment]
Nice af

**Comment by AnthonyJNgo** [comment]
Okay, so you kept the same sprites but just simplified the overall concept? Dope, this will get testmerged.

**Comment by TrickyDickRichard** [comment]
speedmerge now gogogo

**Comment by ghost** [comment]
yes

**Comment by cris5311** [comment]
+1
yes

**Comment by ghost** [comment]
can this get review

**Comment by TrickyDickRichard** [comment]
why hasnt this been speedmerged yet

File: hippiestation/code/modules/crafts/vest.dm [hunk 1]
```diff
@@ -17,4 +17,140 @@
...
+/obj/item/clothing/suit/armor/plate_armor/steel/Initialize(obj/item/I)
+	. = ..()
+	AddComponent(/datum/component/spraycan_paintable)
+	new /obj/item/clothing/head/helmet/plate_armor(loc)
+
+/obj/item/clothing/head/helmet/plate_armor/Initialize()
+	. = ..()
+	AddComponent(/datum/component/spraycan_paintable)
+
+/*/obj/item/clothing/suit/armor/plate_armor/steel/crude
```
Comment (yoyobatty) [inline hippiestation/code/modules/crafts/vest.dm]: Take this entire comment out

File: hippiestation/code/modules/crafts/vest.dm [hunk 2]
```diff
@@ -17,4 +17,140 @@
...
+/datum/crafting_recipe/plate_armor
+	name = "plate armor"
+	result =  /obj/item/clothing/suit/armor/plate_armor/steel
+	reqs = list(/obj/item/stack/sheet/metal = 50,
+				/obj/item/weldingtool)
+	time = 30 //time gating is a shit concept
+	category = CAT_WEAPONRY
+	subcategory = CAT_WEAPON
+
+/*datum/crafting_recipe/plate_armor_plasteel
```
Comment (yoyobatty) [inline hippiestation/code/modules/crafts/vest.dm]: this too

**Review by yoyobatty — DISMISSED** [review]

**Comment by Toxici11i** [comment]
plate armor from forged materials when

**Comment by TrickyDickRichard** [comment]
next year at this rate

**Comment by AnthonyJNgo** [comment]
@zamolxius Make the changes batty requested so I can get this merged.

**Comment by ghost** [comment]
its done

File: hippiestation/code/modules/crafts/vest.dm [hunk 3]
```diff
@@ -17,4 +17,57 @@
...
+	body_parts_covered = CHEST|GROIN|LEGS|FEET|ARMS|HANDS
+	cold_protection = CHEST|GROIN|LEGS|FEET|ARMS|HANDS
+	heat_protection = CHEST|GROIN|LEGS|FEET|ARMS|HANDS
+	allowed = list(/obj/item/flashlight, /obj/item/tank/internals, /obj/item/claymore, /obj/item/forged/melee/sword, /obj/item/forged/melee/dagger, /obj/item/twohanded/forged/greatsword, /obj/item/forged/melee/mace,/obj/item/nullrod)
+	armor = list("melee" = 40, "bullet" = 20, "laser" = -10, "energy" = -10, "bomb" = 15, "bio" = 25, "rad" = 20, "fire" = -15, "acid" = 80)
+	resistance_flags = ACID_PROOF
+	w_class = WEIGHT_CLASS_BULKY
+	strip_delay = 80
+	equip_delay_other = 60
+	slowdown = 0.6
```
Comment (ghost) [inline hippiestation/code/modules/crafts/vest.dm:39]: Does slowdown even work as a float?

File: hippiestation/code/modules/crafts/vest.dm [hunk 4]
```diff
@@ -17,4 +17,57 @@
 				/obj/item/stack/ducttape = 5)
 	time = 80
 	category = CAT_WEAPONRY
-	subcategory = CAT_WEAPON
\ No newline at end of file
+	subcategory = CAT_WEAPON
+
+/obj/item/clothing/suit/armor/plate_armor/steel
```
Comment (ghost) [inline hippiestation/code/modules/crafts/vest.dm:22]: Any particular reason this doesn't have `clothing_flags = THICKMATERIAL` when the helmet does? Intended or oversight?

Comment (ghost) [reply hippiestation/code/modules/crafts/vest.dm:22]: oversight

File: hippiestation/code/modules/crafts/vest.dm (hunk 3 above)
Comment (ghost) [reply hippiestation/code/modules/crafts/vest.dm:39]: you do walk slower in it from what ive seen

**Comment by ghost** [comment]
should be done

File: hippiestation/code/modules/crafts/vest.dm (hunk 1 above)
Comment (ghost) [reply hippiestation/code/modules/crafts/vest.dm]: done

**Comment by ghost** [comment]
Still skeptical of the float part working as intended, never seen anything but integers for that variable. But if it's tested and there's slowdown then sure.

**Review by ghost — APPROVED** [review]
looks okay

**Comment by AnthonyJNgo** [comment]
Going to merge since Mcbawbaggings left a review.

**Pull request merged**